README.md

```diff
@@ -221,17 +221,28 @@ The body simply contains a byte representation of the HTTP payload that will be
 
 [HDFS organizes](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html#The_File_System_Namespace) the data in folders containinig big data files. Such organization is exploited by [`OrionHDFSSink`](doc/design/OrionHDFSSink.md) each time a Flume event is taken from its channel.
 
-Assuming `hdfs_username=myuser`, `service_as_namespace=false` and `attr_persistence=row` as configuration parameters, then the data within the body will be persisted as:
+Assuming `hdfs_username=myuser`, `service_as_namespace=false` and `file_format=json-row` as configuration parameters, then the data within the body will be persisted as:
```

Comment:
Maybe is just a matter of supporting the old `attrs_persistence=row` and `attrs_persistence=column` as "alias" for `file_format=json-row` and `file_format=json-column`. Documentation will not mention `attrs_persistence` anymore, but at least old application using them will still working.